### PR TITLE
[ADD] website_eater: Information about eaters

### PR DESCRIPTION
**Pull request by remytms**
Add the eater information in the website portal.

### CODE REVIEW

**Comment by remytms** [comment]
A review is welcome.

**Comment by tfrancoi** [comment]
Pour moi c'est ok. Y a pas grand chose. 

Après le module website_portal_v10 ca a encore l'air d'être un abération. Si tu veux le portal de la v10 ben migre en v10 quid la maintenance de ce truc ?

File: beesdoo_website_eater/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015-2016 Odoo S.A.
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (houssine78) [inline beesdoo_website_eater/controllers/main.py]: I think you're the only contributor of this code. so remove other people or entity same goes for the other files

File: beesdoo_website_eater/views/beesdoo_website_eater_templates.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+              <t t-esc="eater.name"/>
+            </li>
+          </ul>
+          <span t-if="not eaters">
+            You have no eaters
+          </span>
+        </p>
+      </div>
+    </xpath>
+  </template>
```
Comment (houssine78) [inline beesdoo_website_eater/views/beesdoo_website_eater_templates.xml]: LGTM(Look good to mee)

**Review by houssine78 — APPROVED** [review]
👍

File: beesdoo_website_eater/controllers/main.py (hunk 1 above)
Comment (remytms) [reply beesdoo_website_eater/controllers/main.py]: Done. :)

**Comment by remytms** [comment]
I based my work upon website_portal_v10 because, I think it will be easier to port this module in the future.

Perhaps it's a bad idea,… we will see in the future. ;)

@tfrancoi, @houssine78: Really thank you for your review.

**Pull request closed without merging**